### ISSUE TITLE
Vragrant

### ISSUE THREAD

**Issue by tomjanssens** [timble/www.timble.net#18]
Blog post about Vagrant, inspiration: http://blog.versioneye.com/2015/05/05/vagrant-up

**Comment by magneticmg** [comment]
Hey @tomjanssens . Recently (like yesterday) we've decided not to follow up with the JCM article about vagrant in favor of pursuing a placement on SItePoint. We actually wrote 2 distinct articles about Vagrant for the JCM, the first was too self promotional (apparently). The second submission, which was much more generic and is essentially a overview of how to build your own Joomla vagrant box, just never got a response. Its that second one that you may like to read and we could potential turn that into a Timble post (with some editing and different screenshots)
https://docs.google.com/document/d/1A-v_S-MYs8FucZniexQb1XM2zzvjJl-XnfuPzkjUHsw/edit#heading=h.q1fajmpl54l

Anyway, thought I would give you the back story and make you aware of its existence. @johanjanssens I think we can agree that we're not doing anything else specifically with that piece (confirm)?

**Comment by johanjanssens** [comment]
@magneticmg Agreed. We are not pursuing the second one. Can be re-used elsewhere.
